pkg/agent/flags/flag_controller_option.go

```diff
@@ -0,0 +1,12 @@
+package flags
+
+import "github.com/kolide/launcher/pkg/launcher"
+
+type Option func(*FlagController)
+
+// WithStore sets the key/value store for control data
+func WithCmdLineOpts(cmdLineOpts *launcher.Options) Option {
```

Comment:
Is this basically required? Do you think we'll have more options? (eg: should we just add it to the method signature)